modules/mono/mono_gd/gd_mono.cpp

```diff
@@ -399,8 +404,9 @@ void GDMono::initialize() {
 	}
 
 	if (!is_native_aot) {
-		godot_plugins_initialize = initialize_hostfxr_and_godot_plugins(runtime_initialized);
-		ERR_FAIL_NULL(godot_plugins_initialize);
+		load_assembly_and_get_function_pointer_fn fn = initialize_hostfxr(runtime_initialized);
+		godot_plugins_initialize = initialize_godot_plugins(fn);
+		ERR_FAIL_NULL_MSG(godot_plugins_initialize, "Could not initialize GDMono due to Godot plugins being uninitialized.");
```

Comment:
Not a good error message. This should either be left as it was or replaced with a silent return, since an error is already printed from the initializer function. We already have a custom message when failing to load and get the function pointer for `GodotPlugins` (although it may be better to simplify it). We should add a custom message for when `initialize_hostfxr_for_config`/`initialize_hostfxr_self_contained` fails.